src/components/Input/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+
+import { Wrapper, Label, Input } from './styled'
+
+const InputField = ({ value, label, type = 'text' }) => (
+  <Wrapper>
+    <Label>{label}</Label>
```

Comment:
When you click on this label, your input won't get focused. Label is missing `for` attribute

Reply:
I know about it, but i want keep it for future, when we have some lection with props, but i fix it for now :)